.github/policies/disallow-edits.yml

```diff
@@ -0,0 +1,41 @@
+name: GitOps.PullRequestIssueManagement
+description: GitOps.PullRequestIssueManagement primitive
+resource: repository
+
+configuration:
+  resourceManagementConfiguration:
+    eventResponderTasks:
+      - description: Close PRs that modify files whose "source of truth" is not in this repo.
+        if:
+          - payloadType: Pull_Request
+          - isAction:
+              action: Opened
+          - or:
+              - filesMatchPattern:
+                  pattern: xml/Microsoft.Extensions*/*.xml
+                  excludedFiles:
+                    - xml/Microsoft.Extensions.FileSystemGlobbing/*.xml
+                    - xml/Microsoft.Extensions.FileSystemGlobbing.Abstractions/*.xml
+                    - xml/Microsoft.Extensions.FileSystemGlobbing.Internal*/*.xml
+              - filesMatchPattern:
+                  pattern: xml/System.Composition/*.xml
+              - filesMatchPattern:
+                  pattern: xml/System.Formats.Cbor/*.xml
+              - filesMatchPattern:
+                  pattern: xml/System.Formats.Nrbf/*.xml
+              - filesMatchPattern:
+                  pattern: xml/System.Net.ServerSentEvents/*.xml
+              - filesMatchPattern:
+                  pattern: xml/System.Numerics.Tensors/*.xml
+              - filesMatchPattern:
+                  pattern: xml/System.Speech/*.xml
+              - includesModifiedFile:
+                  file: xml/System.Runtime.Serialization/ISerializationCodeDomSurrogateProvider.xml
+              - includesModifiedFile:
+                  file: xml/System.Runtime.Serialization/ImportOptions.xml
+              - includesModifiedFile:
+                  file: xml/System.Runtime.Serialization/XsdDataContractExporter.xml
```

Comment:
What does includesModifiedFile mean?

Reply:
I was using `filesMatchPattern` for patterns with wildcards, and `includesModifiedFile` for specific files. Still kinda guessing here as the [documentation](https://eng.ms/docs/more/github-inside-microsoft/policies/resource-management?tabs=event-conditions) isn't great.